README.rst

```diff
@@ -31,8 +36,8 @@ The input file is a text file containing person names:
 .. code-block:: console
 
    $ cat inputs/names.txt
-   John Doe
    Jane Doe
+   John Doe
```

Comment:
If you want to amend the order, we could perhaps use different family name:

- Jane Doe
- Joe Bloggs

or even to internationalise the list a bit :wink: 

- Erika Mustermann
- Jan Novak
- Mario Rossi
- ...

See https://en.wikipedia.org/wiki/List_of_placeholder_names_by_language

Reply:
It is just that we are ordering everything alphabetically and it seems a bit weird that those names are inverted.

Reply:
Yup, agree.  It may still be good to demo different family names, perhaps the example may read better?

- Jane Doe
- Joe Bloggs